tests/test_pets.py

```diff
@@ -0,0 +1,137 @@
+import pytest
+
+from src.expected_response_data import responses_from_pet_api as erd
+
+
+@pytest.mark.get_pet
+def test_finding_of_existing_pet(create_pet_api, pc_create_new_pet):
+    api = create_pet_api
+    pet_id = pc_create_new_pet
```

Comment:
when I read a fixture name `pc_create_new_pet` I would expect it to return a pet instance and not just an id. So please update name of a function, or return value